.github/actions/spelling/excludes.txt

```diff
@@ -16,7 +16,15 @@ dotnet-tools.json$
 .htmlvalidate.json$
 appsettings.json$
 package.json$
-package-lock.json$
+package-lock.json
 .psm1$
 .bat$
 .ps1$
+.nuspec.template$
+pnpm-lock.yaml
+.editorconfig
+.DotSettings
+UploadingTestFileXLSX.xlsx
+UploadingTestFileDOCX.docx
+.ruleset
+[Rr]eadme
```

Comment:
Isn't this excluding Readme files? Those are where we have the most documentation, so this shouldn't happen.

Reply:
No, this excludes the WORD readme

Reply:
OK then.

Reply:
Actually this was in the wrong file. Moved it to it's place.